Client/Frontend/Browser/TabsBar/TabsBarViewController.swift

```diff
@@ -106,6 +106,18 @@ class TabsBarViewController: UIViewController {
         updateData()
     }
     
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        
+        let offset = Float(collectionView.contentOffset.x)
+        let startFade = Float(30)
+        leftOverflowIndicator.opacity = min(1, offset / startFade)
+        
+        // all the way scrolled right
+        let offsetFromRight = collectionView.contentSize.width - CGFloat(offset) - collectionView.frame.width
+        rightOverflowIndicator.opacity = min(1, Float(offsetFromRight) / startFade)
```

Comment:
This looks copy pasted from https://github.com/brave/brave-ios/blob/development/Client/Frontend/Browser/TabsBar/TabsBarViewController.swift#L194

Can we move this logic into a shared function and call from both sites?